invenio_search/utils.py

```diff
@@ -43,18 +90,26 @@ def schema_to_index(schema, index_names=None):
     :param index_names: A list of index name.
     :returns: A tuple containing (index, doc_type).
     """
+    warnings.warn(
+        '"schema_to_index" will be moved to invenio-indexer',
```

Comment:
It's better to be explicit here and say `"invenio_search.utils.schema_to_index"`